packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue

```diff
@@ -1,46 +1,115 @@
 <template>
     <div
-        :class="$style['c-breadcrumbs']"
-        data-test-id="breadcrumbs-component">
-        {{ copy.text }}
+        data-test-id="breadcrumbs-component"
+        :class="$style['c-breadcrumbs']">
+        <ul :class="$style['c-breadcrumbs-list']">
+            <template v-for="({ name, url, routerLink }, index) in links">
+                <li
```

Comment:
Accessibility point (can be done in separate PR) – how does this get read out to screen readers? May need an aria attribute to hide this from being read potentially

Reply:
I shall raise a ticket to look into this further in a later PR. 👍